Cargo.toml

```diff
@@ -133,3 +133,6 @@ sqlx = { git = "https://github.com/helium/sqlx.git", rev = "92a2268f02e0cac6fccb
 # [patch.'https://github.com/helium/proto']
 # helium-proto = { path = "../proto" }
 # beacon = { path = "../proto/beacon" }
+# Temporary patch
+[patch."https://github.com/helium/proto"]
```

Comment:
I can try again but last time I set to a specific branch it would not update properly, now that the proto is more settled it might be better. I will attempt.

Reply:
I added this comment. The intent was to patch the proto lib while still developing the definitions. 
To reduce the `update proto -> push proto -> update oracles -> repeat` cycle. 
Then when things are in a good spot, you replace the patch with a pointer to the proto branch.

It might be a good idea to put a comment towards that end in here to help ward off this exact scenario.